api/service/projects_service.go

```diff
@@ -112,72 +118,91 @@ func (service *projectsService) save(project *models.Project) (*models.Project,
 	return service.projectRepository.Save(project)
 }
 
-func (service *projectsService) upsertAuthorizationPolicy(project *models.Project) error {
-	// create administrators policy
-	adminRole, err := service.upsertAdministratorsRole(project)
-	if err != nil {
-		return err
-	}
-	err = service.upsertAdministratorsPolicy(adminRole, project)
-	if err != nil {
-		return err
+func projectReaderRole(project *models.Project) string {
+	return fmt.Sprintf("mlp.projects.%d.reader", project.ID)
```

Comment:
Shall we move the templates to `api/pkg/authz/enforcer/role.go` where the global roles are defined? 

Something like:
```
const (
  ...
  MLPProjectReaderRole = "mlp.projects.{{ .ProjectId }}.reader"
  MLPProjectAdministratorRole = "mlp.projects.{{ .ProjectId }}.administrator"
)
```

The idea is to have the definitions of all of the roles in a single place, which should simplify the understanding of the authZ model for those, who will be maintaining this repo.